src/api/index.ts

```diff
@@ -110,7 +110,13 @@ export class AtelierAPI {
         if (schemas.includes(wsOrFile.scheme)) {
           workspaceFolderName = wsOrFile.authority;
           const parts = workspaceFolderName.split(":");
-          if (parts.length === 2 && config("intersystems.servers").has(parts[0].toLowerCase())) {
+          if (
+            parts.length === 2 &&
+            (config("intersystems.servers").has(parts[0].toLowerCase()) ||
+              vscode.workspace.workspaceFolders.find(
+                (ws) => ws.uri.scheme === "file" && ws.name.toLowerCase() === parts[0].toLowerCase()
```

Comment:
Is it possible that someone would want to do this for non-`file` workspace folders?